tests.py

```diff
@@ -205,3 +210,12 @@ def test_search_s_onelevel(self):
             ('cn=john2,ou=users,dc=30loops,dc=net',{'mail': 'john@example.com', 'userPassword': 'ldaptest'}),
             result
         )
+
+        # search for both johns using anded conditions:
+        result = self.mock_ldap.search_s("ou=groups,dc=30loops,dc=net", ldap.SCOPE_ONELEVEL,
+                                          "(&(objectClass=groupOfUniqueNames)(uniqueMember=cn=john2,ou=users,dc=30loops,dc=net))")
+        print(repr(result))
```

Comment:
A debug print seems to have sneaked in here :smile:

Reply:
oh noes :scream: 

fixed.